stl/inc/compare

```diff
@@ -27,8 +27,25 @@ _STL_DISABLE_CLANG_WARNINGS
 #undef new
 
 _STD_BEGIN
-using _Literal_zero = decltype(nullptr);
-using _Compare_t    = signed char;
+struct partial_ordering;
+struct weak_ordering;
+struct strong_ordering;

```

Comment:
These forward declarations of public types should be `_EXPORT_STD`. (We follow a convention of marking all declarations and definitions `_EXPORT_STD` so we don't need to worry about forgetting it.)